glean-core/android/src/main/java/mozilla/telemetry/glean/net/Upload.kt

```diff
@@ -43,11 +43,18 @@ internal class UploadBody(
     }
 }
 
+@Structure.FieldOrder("tag", "time")
+internal class WaitBody(
+    @JvmField var tag: Byte = UploadTaskTag.Wait.ordinal.toByte(),
+    @JvmField var time: Long = 60_000
+) : Structure() {
+}
+
```

Comment:
Sheesh, what an oversight. Good thing you caught this...

Reply:
I didn't notice in review either. That's why we do the testing.